index.html

```diff
@@ -5,5 +5,15 @@
     <title>Document</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h2>Quiz App</h2>
+    </header>
+    <section>
+      <h3>Very hard question</h3>
+      <h4>show me the answer</h4>
+      <h6 class="tag1">bookmark1</h6>
+      <h6 class="tag2">bookmark2</h6>
```

Comment:
```suggestion
      <h2>Very hard question</h3>
      <h3>show me the answer</h4>
      <span class="tag1">bookmark1</span>
      <span="tag2">bookmark2</span>
```
Since not all of these elements are semantically headlines, I would suggest you maybe a html structure more like that for your cards, or?

Reply:
Thanks for the corrections Feline!